src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+function getFormattedDate(today: Date) {
+  return today.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends React.PureComponent<Props, State> {
+  state: State = {
+    today: new Date(),
```

Comment:
I saw that Dmytro suggested changing the name to "today", but I liked your "date" option better, because we are assigning the date object here, and it seems more logical to me :)